### DISCUSSION TITLE
Hotkey kind of slow

### DISCUSSION THREAD

**Discussion by coffeeneed** [FancyWM/fancywm#259] [General]
Hello,

Is it possible for the developer to speed up the hotkey functionality? When I press the hotkey, it takes 2-3 seconds before I can unfloat my windows since I have to wait for the tooltip to appear at the bottom center.

This is problematic for me since I've wildcarded all windows to unfloat by default, which makes the most sense for me.

Since I need to do this every time I reboot, it's kind of a drag.

Love the software, btw!

**Comment by coffeeneed** [comment]
Sorry, my bad. I found a setting to use without activation hot key. This post can be deleted.

**Comment by veselink1** [comment]
Glad you managed to resolve this. 

But you shouldn't have to wait for the popup to show. It's timing is purposefully separated from the key listener activation, so you should be able to press activation hot key, then F without any delay. 

If this doesn't work, then it is a bug.
